### PR TITLE
dashboard cmd: improve the warning message when kubeconfig is invalid

### PR DESCRIPTION
**Pull request by shorim**
**Description**

Changes proposed in this pull request:

- For the dashboard command, improve the warning message when kubeconfig is invalid

**Related issue(s)**

### CODE REVIEW

File: cmd/kyma/dashboard/cmd.go [hunk 1]
```diff
@@ -112,7 +112,8 @@ func (cmd *command) initContainerRunOpts(envs []string) (docker.ContainerRunOpts
 	kubeconfigPath := kube.KubeconfigPath(cmd.KubeconfigPath)
 	dashboardURL := fmt.Sprintf("http://localhost:%s", cmd.opts.Port)
 	if _, err := kube.NewFromConfig("", kubeconfigPath); err != nil {
-		cmd.CurrentStep.LogInfof("WARNING: kubeconfig is not mounted into container: %v", err)
+		cmd.CurrentStep.LogInfof("WARNING: There was a problem figuring out the current kubeconfig, "+
+			"so the dashboard will start without having the current context attached. More detailed reason:  %v", err)
```
Comment (NHingerl) [inline cmd/kyma/dashboard/cmd.go:116]: Slightly shorter:

```suggestion
			"Thus, the Dashboard will start without the current context. For details, see:  %v", err)
```

File: cmd/kyma/dashboard/cmd.go [hunk 2]
```diff
@@ -112,7 +112,8 @@ func (cmd *command) initContainerRunOpts(envs []string) (docker.ContainerRunOpts
 	kubeconfigPath := kube.KubeconfigPath(cmd.KubeconfigPath)
 	dashboardURL := fmt.Sprintf("http://localhost:%s", cmd.opts.Port)
 	if _, err := kube.NewFromConfig("", kubeconfigPath); err != nil {
-		cmd.CurrentStep.LogInfof("WARNING: kubeconfig is not mounted into container: %v", err)
+		cmd.CurrentStep.LogInfof("WARNING: There was a problem figuring out the current kubeconfig, "+
```
Comment (NHingerl) [inline cmd/kyma/dashboard/cmd.go:115]:
```suggestion
		cmd.CurrentStep.LogInfof("WARNING: Could not identify the current kubeconfig. "+
```

Pls advise whether "identify" is the correct action. I assume this pops up if a) no kubeconfig was provided and b) if it's formatted in a way that we don't recognize automatically.

**Comment by NHingerl** [comment]
There is no .md generated yet? :(

**Comment by shorim** [comment]
@NHingerl This is just a warning which is displayed while the command is running. The `.md` file contains only the description of the command and its flags

File: cmd/kyma/dashboard/cmd.go (hunk 2 above)
Comment (shorim) [reply cmd/kyma/dashboard/cmd.go:115]: yes, that's correct. I will apply your suggestion.

**Review by skhalash — APPROVED** [review]

**Comment by NHingerl** [comment]
✅   too late ;)

**Pull request merged**